arches/app/models/models.py

```diff
@@ -2152,3 +2161,55 @@ def __str__(self):
     class Meta:
         managed = True
         db_table = "spatial_views"
+
+    def clean(self):
+        """
+        Validate the spatial view before saving it to the database as the database triggers have proved hard to test.
+        """
+        super().clean()
+        if self.geometrynode:
+            if self.geometrynode.datatype != "geojson-feature-collection":
+                raise ValidationError(
+                    "Geometry node must be of type geojson-feature-collection"
+                )
```

Comment:
@jacobtylerwalls - If that's the case, I'm wondering if I leave this so that the correct error and message appear until then? I'll need to move super().clean() to the end so the ValidationError is raised first.